README.md

```diff
@@ -0,0 +1,2 @@
+# UMD_Etc
+Repository containing the UMD_Etc code
```

Comment:
@mathomp4 please add this bit: which used for ocean data assimilation

So the one line description would be:
"Repository containing the UMD_Etc code which used for ocean data assimilation"

Reply:
Done. The text that was there was just the default GitHub emits when you ask it to make a README.